wp-content/themes/core/components/blocks/content_loop/css/content_loop.pcss

```diff
@@ -0,0 +1,134 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Content Loop Header
+ *
+ * ----------------------------------------------------------------------------- */
+
+.b-content-loop__header {
+    margin-bottom: var(--spacer-20);
+
+    .c-block__cta-link {
+        margin-top: var(--spacer-10);
+
+        @media (--viewport-medium) {
+            margin-right: var(--spacer-40);
+            margin-top: 0;
+        }
+    }
+
+    @media (--viewport-medium) {
+        display: grid;
+        grid-template-columns: 66.6666% 33.3333%;
+        grid-template-rows: auto;
+        grid-template-areas:
+        "leadin leadin"
+        "title cta"
+        "description cta";
+        margin-bottom: var(--spacer-30);
+        max-width: none;
+    }
+
+    .c-block__leadin {
+        grid-area: leadin;
+    }
+
+    .c-block__title {
+        grid-area: title;
+    }
+
+    .c-block__description {
+        grid-area: description;
+    }
+
+    .c-content-block__cta {
+        grid-area: cta;
+
+        @media (--viewport-medium) {
+            display: flex;
+            align-items: flex-end;
+            justify-content: flex-end;
+        }
+    }
+
+    &.c-content-block {
+        max-width: none;
+    }
+}
+
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Content Loop
+ *
+ * ----------------------------------------------------------------------------- */
+
+.b-content-loop {
+    .c-card__description {
+        margin-top: var(--spacer-30);
+    }
+
+    /* CASE: Columns */
+    &.b-content-loop--layout_columns {
+        .c-card:nth-child(1),
+        .c-card:nth-child(2) {
+            border-top: 1px solid var(--color-grey-light);
+        }
+    }
+
+    /* CASE: Feature */
+    &.b-content-loop--layout_feature {
+        /* hide content block cta on this layout */
+        .c-content-block__cta {
```

Comment:
If the `c-content-block__cta` selector above was un-nested (top-level) than this conditional application (and the one in the header) could both be added as conditions inside the single instance of the cta selector.

Reply:
I agree with this one, but the one in the header feels better to keep next to all the other elements defined in the `grid-tepmlate-areas`